avocado/utils/multipath.py

```diff
@@ -420,3 +421,41 @@ def is_path_added():
     if process.system(cmd) == 0:
         return wait.wait_for(is_path_added, timeout=10) or False
     return False
+
+
+def get_mpath_devices():
+    """
+    Returns wwid, dm_id, disk, disk size, hcil, dm_st, chk_st, dev_st and
+    dev_t from command
+    multipathd -k"show topology"
+
+    :return: dictionary contains mpath values
+    """
```

Comment:
is this different from the one already get_multipath_details ?

Reply:
@maramsmurthy we have all multipath relates methods in multipath.py
and above functionalities are already available.
I suggest if anything missing and you want to add, please append to multipath.py so that all the methods will be at one place.

Reply:
@abdhaleegit get_multipath_details returns details only for specified devices. This definition returns for all the exiting mpath devices. 

This definition i part of multipath.py only

Reply:
@maramsmurthy there is one method which gets you disctionary way of multipath details, please check the utils

Reply:
@abdhaleegit Yes, we have it for one device at one point of time. But we need a definition for all multipath devices at a time. So added this new definition.

Reply:
@maramsmurthy  get_multipath_details():  does not take any one device... it gets all devices ?

Reply:
@maramsmurthy Than I would suggest to enhance the same method instead of new similar util

Reply:
@abdhaleegit used existing definition as suggested. Additional source added to definition to compare even if te dictionary values are dictionary, list or value.